### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by hanahmily**
Fixes https://github.com/apache/skywalking/issues/8989

I leave empty CRUD examples for the future CLI tools.

Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

**Comment by lujiajing1126** [comment]
Where can we preview the docs?

**Comment by hanahmily** [comment]
> Where can we preview the docs?

You can view them locally since they are markdown files. Once this gets merged, I will push them into the official website.

**Comment by wu-sheng** [comment]
We need to adjust the file structure, files like [api/proto/banyandb/measure/v1/docs.md](https://github.com/apache/skywalking-banyandb/pull/111/files#diff-ff8d29cb67ea1f589f6cc82f6261eced8ea3a8bb845eb280d40a11ba1bed2d6a) can't be visualized AFAIk.

**Comment by wu-sheng** [comment]
All markdowns should be hosted inside `/docs`, which is the root of website doc generator.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
 
+The database research community usually uses [RUM conjecture](http://daslab.seas.harvard.edu/rum-conjecture/) to describe how a database access data. In this talk, I will share the details about BanyanDB based on this conjecture, explaining why BanyanDB is more effective and reliable than other storage layers.
+
+## Documents
+
+[Documents](./docs/)
+
+## Resource
+
+- [Data model design](https://docs.google.com/document/d/1XFzHWXSWyIC7AVNIx0ifQ0eJAJYfoPO-2j_lgbl9zu8/edit?usp=sharing)
```
Comment (wu-sheng) [inline README.md:22]: Let's keep the valuable content of this google doc in the markdown as official, and remove this link permanently. I don't want to misguide new contributors/users, and read a version has been changed.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+## Introduction
 
- - [Data model design](https://docs.google.com/document/d/1XFzHWXSWyIC7AVNIx0ifQ0eJAJYfoPO-2j_lgbl9zu8/edit?usp=sharing)
+BanyanDB, as an observability database, aims to ingest, analyze and store Metrics, Tracing, and Logging data. It's designed to handle observability data generated by Apache SkyWalking. Before BanyanDB emerges, the Databases that SkyWalking adopted are not ideal for the APM data model, especially for saving tracing and logging data. Consequently, There’s room to improve the performance and resource usage based on the nature of SkyWalking data patterns.
 
+The database research community usually uses [RUM conjecture](http://daslab.seas.harvard.edu/rum-conjecture/) to describe how a database access data. In this talk, I will share the details about BanyanDB based on this conjecture, explaining why BanyanDB is more effective and reliable than other storage layers.
+
+## Documents
+
+[Documents](./docs/)
```
Comment (wu-sheng) [inline README.md:18]: These should be added later, once website is updated, you should have an official link(s) from released versions to the latest.

**Comment by remGigo** [comment]
> > Where can we preview the docs?
> 
> You can view them locally since they are markdown files. Once this gets merged, I will push them into the official website.

the links in crud package(XXService) seem missing for 404?

File: README.md (hunk 2 above)
Comment (hanahmily) [reply README.md:18]: > All markdowns should be hosted inside `/docs`, which is the root of website doc generator.

done

**Comment by hanahmily** [comment]
> All markdowns should be hosted inside `/docs`, which is the root of website doc generator.

done. Moved all API references into `docs`

File: README.md (hunk 1 above)
Comment (hanahmily) [reply README.md:22]: https://github.com/apache/skywalking/issues/9016 will create update-date documents

File: README.md (hunk 2 above)
Comment (hanahmily) [reply README.md:18]: I will update this once the official documents are published.

**Pull request merged**